src/MetaModels/Helper/ToolboxFile.php

```diff
@@ -933,6 +933,15 @@ private function processFile($fileName)
             }
         }
 
+        // Prepare SVG images.
+        if ($information['isSvgImage'] = $file->isSvgImage) {
+            $information['src'] = $fileName;
+            if (file_exists(TL_ROOT . '/' . $information['src'])) {
+                $size              = getimagesize(TL_ROOT . '/' . $information['src']);
```

Comment:
Why are you fetching the size here and discard the value?
Does this really work with SVG?
If so, we should use the size value returned herin or (if specified) override it with the resize information.

Reply:
that´s right - I confused `size` with `sizetext` at template... I´ll kill this rows